core-bundle/contao/dca/tl_content.php

```diff
@@ -1005,8 +1005,16 @@ public function addCteType($arrRow)
 			$class .= ' empty';
 		}
 
+		$dragHandle = '';
+
+		if (!Input::get('act') && System::getContainer()->get('security.helper')->isGranted(ContaoCorePermissions::DC_PREFIX . $this->strTable, new UpdateAction($this->strTable, $arrRow)))
```

Comment:
```suggestion
		if (!Input::get('act') && System::getContainer()->get('security.helper')->isGranted(ContaoCorePermissions::DC_PREFIX . 'tl_content', new UpdateAction('tl_content', $arrRow)))
```

There is no `$this->strTable` in the `tl_content` DCA.